clients/scheduler/src/index.tsx

```diff
@@ -156,6 +160,8 @@ export const init = ({
       accentColor={accentColor}
       shadowRoot={appRoot.shadowRoot}
       fontFamily={fontFamily}
+      screen={screen}
```

Comment:
think we need `screen` and `timeSlot` added or conditionally added to **ISchedulerProps**